src/shared/components/RealmPanel/RealmPanelAccordion/index.tsx

```diff
@@ -16,6 +19,7 @@ export default function RealmDetailsAccordion({
       hasInteractiveChildren
       openedFromOutside={openedFromOutside}
       closeOpenedFromOutside={closeOpenedFromOutside}
+      additionalTrigger={realmPanelVisible}
```

Comment:
Rename this prop to more readable format, like shouldExpand, or extraExpand, because additionalTrigger says nothing about whether we should expand or collapse it.